packages/components/src/components/tabs/tabs.tsx

```diff
@@ -179,12 +179,12 @@ const Tabs = ({
                             const count = child.props['data-count'];
                             return (
                                 <Tab
-                                    active_icon_color={active_icon_color}
+                                    active_icon_color={active_icon_color ?? ''}
                                     className={className}
                                     count={count}
                                     icon={icon}
-                                    icon_color={icon_color}
-                                    icon_size={icon_size}
+                                    icon_color={icon_color ?? ''}
+                                    icon_size={icon_size ?? 0}
```

Comment:
You should set the fallback as a default value in the args of the component. Easier to track and more readable that way

Reply:
Will do, thank you